app/javascript/widget/components/UnreadMessageList.vue

```diff
@@ -27,7 +27,12 @@
         class="button clear-button"
         @click="openConversationView"
       >
-        <span class="flex items-center">
+        <span
+          class="flex items-center"
+          :style="{
+            color: widgetColor,
```

Comment:
One problem I can see is that if the user selected white, the button would become invisible.

Reply:
@pranavrajs Is is fine? 
<img width="225" alt="image" src="https://user-images.githubusercontent.com/64252451/206360160-9492448a-2a1d-436e-9d7e-1e8adb813b17.png">

https://github.com/chatwoot/chatwoot/pull/6033/commits/3e06cb6d928fdf5b1d2a81fabbb50efa2792ff60